dataset/src/test/scala/frameless/functions/NonAggregateFunctionsTests.scala

```diff
@@ -17,6 +17,80 @@ class NonAggregateFunctionsTests extends TypedDatasetSuite {
     super.afterAll()
   }
 
+  test("sha2") {
+    val spark = session
+    import spark.implicits._
+
+    def prop(values:List[X1[Array[Byte]]])(implicit encX1:Encoder[X1[Array[Byte]]]) = {
```

Comment:
Can you add spaces after the `:`? (Same for the other two tests)

Reply:
ok